scripts/build.js

```diff
@@ -1,19 +1,25 @@
 const { execSync } = require('child_process');
-const services = require(`${__dirname}/toBuild.json`);
+//FIXME have no access to integration-content-repository. Temporary exclude it from build
```

Comment:
That will remove all Services from the build

Reply:
If you want to remove icr from build make a condition in the check for changes script!

Reply:
#93 is fixed already, but the dockerfile needs to be reworkt in icr.

Latest is only set on master builds. Because build and deploy only hits master branches.

I will review the changes tomorrow. Perhabs i missed something on my mobil ;) will come back on it

Reply:
@falkvoigt #93 is open yet and it's blocking this pr. What can we do to proceed?

Reply:
we need to merge the changes @hschmidthh did in his Branch an it's fine. @drobiazko please have a look at my comment below. there are some changes you did in the repo which needs some attention.